controller/src/main/java/org/jboss/as/controller/AttributeParsers.java

```diff
@@ -200,6 +198,7 @@ public void parseElement(AttributeDefinition attribute, XMLExtendedStreamReader
             if (attribute.getXmlName().equals(reader.getLocalName())) {
                 ObjectTypeAttributeDefinition objectType = ((ObjectTypeAttributeDefinition) attribute);
                 ModelNode op = operation.get(attribute.getName());
+                op.setEmptyObject();
```

Comment:
I guess it's not so strange; this is a parser, so it expects to be starting with a clean operation node.

Was this showing up as a bug somehow or did you add this to be thorough?